Makefile

```diff
@@ -188,13 +194,15 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 build-image: ko ## Build the kro controller images using ko build
 	echo "Building kro image $(RELEASE_VERSION).."
 	$(WITH_GOFLAGS) KOCACHE=$(KOCACHE) KO_DOCKER_REPO=$(KO_DOCKER_REPO) \
+		GIT_COMMIT=$(GIT_COMMIT) RELEASE_VERSION=$(RELEASE_VERSION) BUILD_DATE=$(BUILD_DATE) \
 		$(KO) build --bare github.com/kro-run/kro/cmd/controller \
 		--local\
 		--push=false --tags ${RELEASE_VERSION} --sbom=none
 
 .PHONY: publish
 publish-image: ko ## Publish the kro controller images to ghcr.io
-	$(WITH_GOFLAGS) KOCACHE=$(KOCACHE) \
+	$(WITH_GOFLAGS) KOCACHE=$(KOCACHE) KO_DOCKER_REPO=$(KO_DOCKER_REPO) \
```

Comment:
Is the fact the `KO_DOCKER_REPO` wasn't here before a bug?

Reply:
Correct, i was testing it with a custom OCI registry, looks like this is needed